components/Canvas/filter/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
```

Comment:
the name of folder should be "Filter"?

Reply:
was it best practice to name folder in Uppercase convention?